wrappers/map.js

```diff
@@ -117,20 +119,42 @@ class Map extends EventEmitter {
 
     this.geojsonLayers[name] = layer
 
+    this.sortLayers()
+
     map.emit('geojson.add', {
       name: name,
       layer: layer
     })
   }
 
+  sortLayers() {
+    var sortableLayers = []
+    Object.keys(this.geojsonLayers).forEach((name) => {
+      var layer = this.geojsonLayers[name]
+      if (!isNaN(parseInt(nn(layer)('options.sortOrder').val, 10))) {
+        sortableLayers.push(layer)
+      }
+    })
+    sortableLayers.sort(function (a, b) {
+      return a.options.sortOrder - b.options.sortOrder
+    })
+    sortableLayers.forEach(function (layer) {
+      try {
+        layer.bringToFront()
+      } catch (e) {}
+    })
```

Comment:
Oh, sorry, forgot about that. Where is `R.values` when you need it?? What about `Object.keys(obj).map(key => obj[key]).filter().sort().etc` ?